### DISCUSSION TITLE
Renovate not running on renamed GH repo

### DISCUSSION THREAD

**Discussion by ripecosta** [renovatebot/renovate#23720] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the problem

I renamed two GH repositories that were running Renovate. After the rename Renovate skips the repos.
I tried to rename the onboarding PR and removing the `renovate.json` file but nothing changed.

Renovate's App Dashboard doesn't list the new repo name. The dashboard page for the new repo name can be accessed directly through `https://developer.mend.io/github/<org>/<repo-name>` but it list the same exact jobs as the page for the old repo name does. Triggering a run from either of the pages results in Renovate running on the old repo name and skipping it.

### Relevant debug logs

<details><summary>Logs</summary>

Org/repo names obfuscated

```
INFO: Repository started
{
  "renovateVersion": "36.27.1"
}

DEBUG: Using localDir: /tmp/worker/43794e/1ba294/repos/github/MyOrg/old-repo-name
DEBUG: PackageFiles.clear() - Package files deleted
DEBUG: initRepo("MyOrg/old-repo-name")
DEBUG: Repository has been renamed
{
  "desiredRepo": "MyOrg/old-repo-name"
  "foundRepo": "MyOrg/new-shiny-repo-name"
}

DEBUG: Caught initRepo error
{
  "err": {
    "message": "renamed",
    "stack": "Error: renamed\n    at Proxy.initRepo (/opt/containerbase/tools/renovate/36.27.1/node_modules/renovate/lib/modules/platform/github/index.ts:434:13)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at getPlatformConfig (/opt/containerbase/tools/renovate/36.27.1/node_modules/renovate/lib/workers/repository/init/apis.ts:57:26)\n    at initApis (/opt/containerbase/tools/renovate/36.27.1/node_modules/renovate/lib/workers/repository/init/apis.ts:69:12)\n    at initRepo (/opt/containerbase/tools/renovate/36.27.1/node_modules/renovate/lib/workers/repository/init/index.ts:42:12)\n    at Object.renovateRepository (/opt/containerbase/tools/renovate/36.27.1/node_modules/renovate/lib/workers/repository/index.ts:54:14)\n    at attributes.repository (/opt/containerbase/tools/renovate/36.27.1/node_modules/renovate/lib/workers/global/index.ts:184:11)\n    at start (/opt/containerbase/tools/renovate/36.27.1/node_modules/renovate/lib/workers/global/index.ts:169:7)\n    at /opt/containerbase/tools/renovate/36.27.1/node_modules/renovate/lib/renovate.ts:18:22"
  }
}

INFO: Repository has been renamed - skipping
DEBUG: Repository result: renamed, status: disabled, enabled: false, onboarded: undefined
```

</details>

### Have you created a minimal reproduction repository?

Placeholder value, please select the correct response from the dropdown

**Comment by PhilipAbed** [comment, marked as answer]
Hi, We are working on the issue. 

as for a workaround: 
please go to [renovate app](https://github.com/apps/renovate) =>...sign in your org...=> configure => do any change to the selected repositories

**Reply by PhilipAbed** [reply]
here's a screenshot
![image](https://github.com/renovatebot/renovate/assets/42116482/b7bdc9ba-04b7-4d39-8000-527daee96c02)

any change to the list should update your renamed repositories

**Reply by ripecosta** [reply]
Thanks Phil. Your suggestion fixed it.

We have Renovate enabled on all repositories. It would be great if Renovate could pick up these things and initialise on the renamed repo the same way that is does when we create a new repo. Is there a plan to implement this? Or to implement a way to trigger a repo list refresh (maybe from the App Dashboard?) that doesn't require Org Owner perms?

**Reply by PhilipAbed** [reply]
it won't require any action from the user.
After the fix we are working on, the next time you rename a repo, it will be installed automatically.

**Comment by theurichde** [comment]
Hi @PhilipAbed 👋🏻  

I am facing the exactly same issue today. Is the fix you were working on already implemented and live?
